src/main/java/codeu/model/data/User.java

```diff
@@ -79,6 +65,15 @@ public Instant getCreationTime() {
     return creation;
   }
 
+  /** Returns whether User is admin or not. */
+  public Boolean getIsAdmin() {
+    return isAdmin;
+  }
+  /** Changes admin status of User */
+  public void setAdmin() {
+    this.isAdmin = !(this.isAdmin)  ;
```

Comment:
This method seems more like `invertAdminStatus`. Either change the name or supply and argument.